.gitignore

```diff
@@ -1,2 +1,3 @@
 _build/
 Gemfile.lock
+.idea
```

Comment:
`.idea` is a directory used by JetBrains products (like WebStorm) to store configuration files. It imho does not hurt to have it here, and it is convenient if you work with WebStorm to have the files ignored, so they are not added by accident.